src/content/docs/en/core-concepts/astro-syntax.mdx

```diff
@@ -148,6 +148,41 @@ In Astro, you use the standard `kebab-case` format for all HTML attributes inste
 <div class="box" data-value="3" />
 ```
 
+#### Node in Script Body
+
+In Astro, you can't use JSX node in script body.
+
+```astro title="TodoList.astro" "<TodoItem todo={todo} />"
+---
+import TodoItem from "./TodoItem.astro"
+const todos = [
+  { id: 1, desc: "Todo 1" },
+  { id: 2, desc: "Todo 2" }
+]
+
+// 🚫 This doesn't work
+const renderedTodos = todos.map(todo => <TodoItem todo={todo} />) 
+---
+
+<div>
+  {renderedTodos}
+</div>
+```
+
+#### Node as Property Input
+
+In Astro, you can't use component or html nodes as property input.
```

Comment:
Perhaps instead of "Property Input", we can refer to them as "Attribute Values", similar to the "Attributes" section above. Maybe this can be merged into the "Attributes" section as well, but I'm also fine if it's a new section.